flow360/component/flow360_params/flow360_output.py

```diff
@@ -0,0 +1,323 @@
+    write_single_file: Optional[bool] = pd.Field(alias="writeSingleFile")
+    primitive_vars: Optional[bool] = pd.Field(alias="primitiveVars")
+    start_average_integration_step: Optional[bool] = pd.Field(alias="startAverageIntegrationStep")
+    output_fields: Optional[List[Union[CommonFieldVars, SurfaceFieldVars]]] = pd.Field(
+        alias="outputFields"
+    )
+    surfaces: Optional[Surfaces] = pd.Field()
+
+    Cf: Optional[bool] = pd.Field(alias="Cf")
+    Cf_vec: Optional[bool] = pd.Field(alias="CfVec")
+    Cf_normal: Optional[bool] = pd.Field(alias="CfNormal")
+    Cf_tangent: Optional[bool] = pd.Field(alias="CfTangent")
+    y_plus: Optional[bool] = pd.Field(alias="yPlus")
+    wall_distance: Optional[bool] = pd.Field(alias="wallDistance")
+    heat_flux: Optional[bool] = pd.Field(alias="heatFlux")
+    node_forces_per_unit_area: Optional[bool] = pd.Field(alias="nodeForcesPerUnitArea")
+    node_normals: Optional[bool] = pd.Field(alias="nodeNormals")
+
+
+class SurfaceOutputPrivate(SurfaceOutput):
+    """:class:`SurfaceOutputPrivate` class"""
+
+    wall_function_metric: Optional[bool] = pd.Field(alias="wallFunctionMetric")
+    node_moments_per_unit_area: Optional[bool] = pd.Field(alias="nodeMomentsPerUnitArea")
+    residual_sa: Optional[bool] = pd.Field(alias="residualSA")
+    coarsen_iterations: Optional[int] = pd.Field(alias="coarsenIterations")
+
+
+class Slice(Flow360BaseModel):
+    """:class:`NamedSlice` class"""
+
+    slice_normal: Coordinate = pd.Field(alias="sliceNormal")
+    slice_origin: Coordinate = pd.Field(alias="sliceOrigin")
+    output_fields: Optional[List[str]] = pd.Field(alias="outputFields")
+
+
+class Slices(Flow360SortableBaseModel):
+    """:class:`SelfNamedSlices` class"""
+
+    @classmethod
+    def get_subtypes(cls) -> list:
+        return [_GenericSliceWrapper.__fields__["v"].type_]
+
+    # pylint: disable=no-self-argument
+    @pd.root_validator(pre=True)
+    def validate_slice(cls, values):
+        """
+        root validator
+        """
+        return _self_named_property_validator(
+            values, _GenericSliceWrapper, msg="is not any of supported slice types."
+        )
+
+
+class _GenericSliceWrapper(Flow360BaseModel):
+    """:class:`_GenericMonitorWrapper` class"""
+
+    v: Slice
+
+
+class SliceOutput(OutputBase):
+    """:class:`SliceOutput` class"""
+
+    output_format: Optional[OutputFormat] = pd.Field(alias="outputFormat")
+    animation_frequency: Optional[Union[PositiveInt, Literal[-1]]] = pd.Field(
+        alias="animationFrequency"
+    )
+    animation_frequency_offset: Optional[int] = pd.Field(alias="animationFrequencyOffset")
+    output_fields: Optional[List[Union[CommonFieldVars, VolumeSliceFieldVars]]] = pd.Field(
+        alias="outputFields"
+    )
+    slices: Optional[Slices]
+
+    bet_metrics: Optional[bool] = pd.Field(alias="betMetrics")
+    bet_metrics_per_disk: Optional[bool] = pd.Field(alias="betMetricsPerDisk")
```

Comment:
move all bool options to separate class that will be only used in updater but not be part of allowed configuration

Reply:
Fixed in 05ca5e6